src/index.ts

```diff
@@ -193,7 +193,13 @@ const canWriteSomeProgram = (permissions: string[]) => {
  * @param args
  */
 const isProgramAdmin = (args: { permissions: string[]; programId: string }): boolean =>
-  canWriteProgram(args);
+  args.permissions.some(
+    code =>
+      code &&
+      code.startsWith(PROGRAM_PREFIX) &&
+      code.includes(PERMISSIONS.WRITE) &&
+      code.includes(args.programId),
+  );
```

Comment:
I think it would be better to directly check for the exact string we expect
```suggestion
  args.permissions.some(
    code => code === `${PROGRAM_PREFIX}-${args.programId}.${PERMISSIONS.WRITE}`
  );
```

Reply:
Suggesting the change because the proposed check makes it possible for some unexpected alternate permissions to pass the check.

Reply:
Muy bueno! Updated